resources/conandata.yml

```diff
@@ -0,0 +1 @@
+version: "5.8.0-alpha.0"
```

Comment:
Should this be committed?

Reply:
Yes, previously the `cura_resources` package used the `conandata.yml` from Cura, which is easier for testing but not for integration. Now it has its own file.